Resources/services/importers.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="AdyenPayment\Import\PaymentMethodImporter">
+            <argument type="service" id="AdyenPayment\Components\Adyen\PaymentMethod\PaymentMethodsProvider"/>
+            <argument type="expression">service('models').getRepository('Shopware\\Models\\Shop\\Shop')</argument>
+            <argument type="service" id="AdyenPayment\Rule\AdyenApi\UsedFallbackConfigRule" />
+            <argument type="service" id="AdyenPayment\Components\Adyen\Mapper\PaymentMethodMapper" />
+            <argument type="service" id="AdyenPayment\Doctrine\Writer\PaymentMethodWriter"/>
+            <argument type="service" id="models"/>
+        </service>
+<!--        <service id="AdyenPayment\Import\TraceablePaymentMethodImporter"-->
```

Comment:
![shock](https://media.giphy.com/media/PqcIFm93VxA8o/giphy.gif)

you actually completely disabled the logging for import